### PR TITLE
New package: SABnzbdTeam.SABnzbd.Portable version 4.2.3

### PR DESCRIPTION
**Pull request by CoolPlayLin**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/150298)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-04-21T00:27:52.5045081Z ##[error]Manifest Validation Failed
 2024-04-21T00:27:57.5305661Z ##[section]Finishing: Validate Manifest

(Automated response - build 877.)

File: manifests/s/SABnzbdTeam/SABnzbd/Potable/4.2.3/SABnzbdTeam.SABnzbd.Potable.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: SABnzbdTeam.SABnzbd.Potable
```
Comment (stephengillie) [inline manifests/s/SABnzbdTeam/SABnzbd/Potable/4.2.3/SABnzbdTeam.SABnzbd.Potable.yaml:4]:
```suggestion
PackageIdentifier: SABnzbdTeam.SABnzbd.Portable
```
Please update file name and path to match this change.

File: manifests/s/SABnzbdTeam/SABnzbd/Potable/4.2.3/SABnzbdTeam.SABnzbd.Potable.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- RelativeFilePath: SABnzbd-4.2.3/SABnzbd.exe
+UpgradeBehavior: install
+ReleaseDate: 2024-03-11
+Installers:
+- Architecture: x86
+  NestedInstallerType: potable
+  InstallerUrl: https://github.com/sabnzbd/sabnzbd/releases/download/4.2.3/SABnzbd-4.2.3-win32-bin.zip
+  InstallerSha256: E486C0E82791919A65ED7E57C25E589BA7D5E8BCA8F6C38E77E3C8A75F4FB5B6
+- Architecture: x64
+  NestedInstallerType: potable
```
Comment (stephengillie) [inline manifests/s/SABnzbdTeam/SABnzbd/Potable/4.2.3/SABnzbdTeam.SABnzbd.Potable.installer.yaml:18]:
```suggestion
  NestedInstallerType: portable
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @CoolPlayLin,

It seems there might have been a typo. This might not have been caught by the spelling check because potable `fit to drink` is still a valid English word.

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

**Comment by Trenly** [comment]
@CoolPlayLin - Support for zipped binaries has now been added and is available in the pipelines. This will require you to update your submission to ManifestVersion: 1.9.0 and set ArchiveBinariesDependOnPath: true in the installer node. To keep things clean, this PR is being closed. Please submit a new PR for the latest version of this package. Thank you for your contributions!

**Pull request closed without merging**